src/web/handlers/GestureHandler.ts

```diff
@@ -736,6 +737,14 @@ export default abstract class GestureHandler {
     return false;
   }
 
+  protected wasPressedButtonCorrect(mouseButton: MouseButton | undefined) {
+    return (
+      !mouseButton ||
```

Comment:
Left mouse button is `0` which is falsy

Reply:
At this point left button was already mapped to value `1` in `PointerEventManager`

```tsx
this.mouseButtonsMapper.set(0, MouseButton.LEFT);
```